### ISSUE TITLE
F1 score can not match the paper

### ISSUE THREAD

**Issue by ghost** [benedekrozemberczki/SGCN#9]
It seems that F1 score can not match the paper, for example, I set epoch=200, layers=96 64 32, learning rate=0.001, but on BIT-OTC dataset the F1 score can only reach 0.802. 

I was wondering if there is anything wrong about my experimental settings?

**Comment by benedekrozemberczki** [comment]
What is your cutoff?

**Comment by songsong0425** [comment]
> 

I faced the same issue while running the code without any modification.
Can you describe the hyperparameters for reproducing the performance in the paper? (i.e. Table 2 in IEEE paper and Table 3,4 in arxiv paper)

Also, I found a mismatch between model options in Readme file and param_parser file.
In Readme file, the reduction interactions and dimension were set as 128, 30, respectively.
But those parameters were set as 30 and 64 in the parameter parser.
Which one should I follow for the final result?
